src/components/RegistrationForm/Form/Form.styled.js

```diff
@@ -2,34 +2,29 @@ import styled from 'styled-components';
 import { device } from '../../../utils';
 
 export const StyledForm = styled.form`
+  margin-bottom: 17px; // відступ між формою та текстом нижче
```

Comment:
// margin between the form and text below.

Reply:
change text